test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -347,17 +362,22 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 				prefix = 'coblocks-';
 				blockClass = 'gallery-masonry';
 				break;
+			case 'Premium Content':
+				blockClass = 'premium-content-container';
+				break;
 		}
 
 		const selectorAriaLabel = ariaLabel || `Block: ${ title }`;
 
+		// TODO Remove the `deprecatedInserterBlockItemSelector` definition and usage after we activate GB 9.4.0 on production.
+		const deprectedInserterBlockItemSelector = `.edit-post-layout__inserter-panel .block-editor-inserter__block-list button.editor-block-list-item-${ prefix }${ blockClass }`;
 		const inserterBlockItemSelector = By.css(
-			`.edit-post-layout__inserter-panel .block-editor-inserter__block-list button.editor-block-list-item-${ prefix }${ blockClass }`
+			`.edit-post-layout__inserter-panel .block-editor-block-types-list button.editor-block-list-item-${ prefix }${ blockClass }, ${ deprectedInserterBlockItemSelector }`
 		);
 
 		let insertedBlockSelector = By.css(
 			`.block-editor-block-list__block.${
-				hasChildBlocks ? 'has-child-selected' : 'is-selected'
+				initsWithChildFocus ? 'has-child-selected' : 'is-selected'
```

Comment:
Do we still need this ternary if the `initsWithChildFocus` is now constant? Also, is it backwards compatible?

Reply:
Oh, nice! TIL 👍🏻

Reply:
Making it a `const` was an oversight, it should be kept being defined with a `let`. Thanks for catching this!

Reply:
It's not breaking anything as it's not being used anymore by any of the blocks, for now. It was needed for the Jetpack blocks, but they seem to have changed recently and are not giving auto focus to the first inner child upon being initialized.

Reply:
We might even be able to remove it 🤔

Reply:
I'll let it as a `let` for now as I'm unsure whether other blocks still use (or will end up using) the `has-child-selected` CSS class.

Reply:
The `const` was actually added by the linter, which won't let me keep it as `let` as long as it's not re-assigned, which is to be expected. I didn't want to remove it as it servers as a hint that there might be blocks we add in the future (or that end up changing its behavior) that will use this CSS class. 

What I did instead was to refactor it a bit to make it a bit easier to change/maintain and reason about, see https://github.com/Automattic/wp-calypso/pull/47533. I'll rebase this PR later when I confirm everything's working as expected after this change.